### PR TITLE
Bugfix/6044/disable voice recording on readonly conversations

### PR DESCRIPTION
**Pull request by marcoambrosini**
fix: #6044

### CODE REVIEW

File: src/components/NewMessageForm/AudioRecorder/AudioRecorder.vue [hunk 1]
```diff
@@ -327,6 +346,18 @@ export default {
 				opacity: 1;
 			}
 		}
+		&--disabled {
```
Comment (PVince81) [inline src/components/NewMessageForm/AudioRecorder/AudioRecorder.vue:349]: any chance to make this global in some way ?
how are we doing it for the other buttons like the attachment one ?

Comment (marcoambrosini) [reply src/components/NewMessageForm/AudioRecorder/AudioRecorder.vue:349]: It'll be included in the button component, which I will pick up again soon!

**Comment by PVince81** [comment]
I think it's not transparent enough, see if you compare it with the attachment icon:
image

Now it seems the attachment button might be an action icon while the voice recording one isn't:
image
(the hover is there even when disabled, it's a separate issue)

**Comment by marcoambrosini** [comment]
@PVince81 I darkened the attachment Icon which was way too light. The emoji button behaves differently as its opacity increases on hover. I will standardise everything in the button component.

**Review by PVince81 — APPROVED** [review]
:+1:

**Pull request merged**